nautobot/docs/release-notes/version-1.1.md

```diff
@@ -2,48 +2,61 @@
+1. `execute_query()`: Runs string as a query against GraphQL.
+2. `execute_saved_query()`: Execute a saved query from Nautobot database.
 
-#### MySQL Database Support [#17](https://github.com/nautobot/nautobot/issues/17)
+#### MySQL Database Support ([#17](https://github.com/nautobot/nautobot/issues/17))
 
 MySQL 8.x is now fully supported as a database backend!
 
 The installation and configuration guides have been revised to include MySQL. If you prefer MySQL or it is more easily supported in your environment, configuring Nautobot to use MySQL is as easy as changing value of `ENGINE` in your `DATABASES` setting to point to `django.db.backends.mysql` and installing the MySQL Python driver using `pip3 install nautobot[mysql]`.
 
+!!! note
+    You will not be able to directly migrate your data from PostgreSQL to MySQL. A fresh start is required.
+
 A new `NAUTOBOT_DB_ENGINE` environment variable has been added to allow for specifying the desired database engine at runtime without needing to modify your `nautobot_config.py`. Please see the [configuration guide on `DATABASES`](../configuration/required-settings.md#databases) for more details on how to configure Nautobot to use MySQL.
 
 Please see the MySQL setup guides for [Ubuntu](../installation/ubuntu.md#mysql-setup) and [CentOS](../installation/centos.md#mysql-setup) to get started.
 
-#### Read Only Jobs [#200](https://github.com/nautobot/nautobot/issues/200)
+#### Read Only Jobs ([#200](https://github.com/nautobot/nautobot/issues/200))
 
 Jobs may be optionally marked as read only by setting the `read_only = True` meta attribute. This prevents the job from making any changes to nautobot data and suppresses certain log messages. Read only jobs can be a great way to safely develop new jobs, and for working with reporting use cases. Please see the [Jobs documentation](../additional-features/jobs.md) for more details.
 
-#### Config Context Schemas [#274](https://github.com/nautobot/nautobot/issues/274)
+#### Config Context Schemas ([#274](https://github.com/nautobot/nautobot/issues/274))
 
 While config contexts allow for arbitrary data structures to be stored within Nautobot, at scale it is desirable to apply validation constraints to that data to ensure its consistency and to avoid data entry errors. To service this need, Nautobot supports optionally backing config contexts with [JSON Schemas](https://json-schema.org/) for validation. These schema are managed via the config context schema model and are optionally linked to config context instances, in addition to devices and virtual machines for the purpose of validating their local context data. Please see the [docs](../additional-features/config-contexts.md#config-context-schemas) for more details.
 
-#### Plugin Defined Navigation [#12](https://github.com/nautobot/nautobot/pull/572)
+#### Background Tasks now use Celery ([#223](https://github.com/nautobot/nautobot/issues/223))
```

Comment:
I'm not sure what order we're following in this document now - I think we'd started off trying to do alphabetical order of subheadings, but it looks like that's fallen away. Any thoughts here?

Also, thoughts on whether this one belongs better under `Added` or under `Changed`?

Reply:
I think having them in alpha is a good idea. I will sort them.

And yeah. Changed for that one.